frontend_react/src/container/About.jsx

```diff
@@ -13,31 +13,42 @@ const { react, js, next, sanity, mongodb, tailwind } = images;
 // ];
 
 const techs = [
-  { title: 'ReactJS', imgUrl: react },
-  { title: 'NextJS', imgUrl: next },
-  { title: 'Javascript', imgUrl: js },
-  { title: 'Tailwind', imgUrl: tailwind },
-  { title: 'Sanity', imgUrl: sanity },
-  { title: 'MongoDB', imgUrl: mongodb },
+  { title: 'ReactJS', imgUrl: react, link: 'https://reactjs.org/' },
+  { title: 'NextJS', imgUrl: next, link: 'https://nextjs.org/' },
+  { title: 'Javascript', imgUrl: js, link: 'https://www.javascript.com/' },
+  { title: 'Tailwind', imgUrl: tailwind, link: 'https://tailwindcss.com/' },
+  { title: 'Sanity', imgUrl: sanity, link: 'https://www.sanity.io/' },
+  { title: 'MongoDB', imgUrl: mongodb, link: 'https://www.mongodb.com/' },
 ];
 
 function About() {
   return (
     <div className="border border-red-500 ">
-      <div className="border border-red-200 flex  flex-col pt-[50px] md:flex-row md:pt-[110px] px-[200px] justify-between items-center font-medium text-[42px]">
-        <div className="border border-yellow-300">
-          <div className="flex w-[86px] h-0.5 bg-primary" />
+      <div className="flex flex-col pt-[50px] md:flex-row md:pt-[110px] md:px-[200px] justify-evenly items-center gap-10 md:gap-20">
+        <div className="flex flex-col md:flex-row items-center flex-1">
+          <div className="flex w-[86px] h-0.5 bg-primary mb-2" />
           <div className="flex text-[28px] md:text-[42px] gap-2">
             <span>My</span>
             <br />
             <span>Technologies</span>
           </div>
         </div>
-        <div className="border border-yellow-300 flex md:flex-row justify-between gap-3">
+        <div className="flex md:flex-row justify-between gap-2 flex-1 pt-">
           {techs.map((tech) => (
-            <div className="text-xs">{tech.title}
-              <img className="shadow-md shadow-primary w-10 h-10 object-cover flex-1 items-center rounded-full" src={tech.imgUrl} />
-            </div>
+            <motion.a
```

Comment:
Missing key prop. It's necessary.

If you haven't read, check this article on [why key prop is important in React.js](https://epicreact.dev/why-react-needs-a-key-prop/)